vuln/npm/389.json

```diff
@@ -0,0 +1,21 @@
+{
+  "id": "389",
+  "title": "Information Exposure Through Directory Listing",
+  "overview": "[serve] Directory listing and File access even when they have been set to be ignored.",
+  "created_at": "2018-01-24",
+  "updated_at": "2018-03-14",
+  "publish_date": "2018-03-13",
+  "author": "digitalwizard",
+  "module_name": "serve",
+  "cves": [
+    "CVE-2018-3718"
+  ],
+  "vulnerable_versions": "<=6.4.9",
+  "patched_versions": ">=6.5.2",
```

Comment:
Does the version really jump from 6.4.9 to 6.5.2, or what?

Reply:
No, you're right, it should be <6.5.2
The original report was for 6.4.9